src/styles/header.scss

```diff
@@ -0,0 +1,119 @@
+.header {
+  @include desktop {
+    max-width: 1280px;
+  }
+
+  @include big-desktop {
+    max-width: 1440px;
+
+  }
+
+  &__nav {
+    padding: 15px 0 20px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    @include tablet {
+      padding: 20px 0 30px;
+    }
+
+    @include desktop {
+      max-width: 1280px;
+      margin: auto;
+      padding: 50px 0 90px;
+    }
+  }
+
+  &__menu {
+    @include desktop {
+      display: none;
+    }
+  }
+
+  &__logotipe {
+    display: none;
+
+    @include desktop {
+      display: flex;
+      align-items: center;
+    }
+  }
+
+  &__logo {
+    @include desktop {
+      display: none;
+    }
+  }
+
+  &__bar {
+    display: flex;
+    align-items: center;
+    gap: 292px;
```

Comment:
The fixed gap between flex items might cause an issue on different screen sizes. It's better to use responsive units like percentages for gaps.